src/exceptions/HO100322.ts

```diff
@@ -0,0 +1,9 @@
+import type { ExceptionGenerator } from "src/types/ExceptionGenerator"
+
+const HO100322: ExceptionGenerator = (hearingOutcome) => {
```

Comment:
Happy to do these exceptions in a second pass - please could you add tickets to make sure we don't forget?

Reply:
I was actually going to do the exceptions in my second PR as part of the same ticket. The logic is relevant to the amendment and should be easier to implement now.